### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by erindcole** [erindcole/mentalhues#5]
#General Ideas:
### Overall, the logo needs to evoke positive emotions. Bright colors, a warm image, and things that generally make a person smile are included.
#Specifications:
###1. Include a representation of a spectrum of a certain color. Pick one color, and show its spectrum in the image.
###2. Some representation of mental health or mental health disorders.
###3. The text "Mental Hues"

**Comment by erindcole** [comment]
@yochannah, a fellow Mozilla Open Leader, received an amazing logo during MozFest 2017 for her project @yochannah/code-is-science! So awesome, right?! I want one too!

**Comment by yochannah** [comment]
@erindcole A couple of ways you can glean interest in your tickets is by using projects like [First Timers Only](http://www.firsttimersonly.com/) and [Up for Grabs](https://github.com/up-for-grabs/up-for-grabs.net). They're both projects designed to help people who want to contribute to open source, but need guidance to make their first pull request. 

First timers only is pretty easy - just tag a github issue with first-timers-only and make sure the issue is pretty clear, with step-by-step descriptions (e.g. how to pick up the issue, what file format you want, what folder to put it in the repo). Once you've tagged the issue, it'll also get tweeted by the first timers only bot - https://twitter.com/first_tmrs_only?lang=en 

Up for grabs takes a little more effort to include your project - I'd read through the readme first.

You could also add the help-wanted label so people can find it this way: https://github.com/search?q=label%3Ahelp-wanted&state=open&type=Issues&utf8=%E2%9C%93

**Comment by erindcole** [comment]
@yochannah thanks for the suggestion! I plan to include labels in my Contributing Guidelines, and it would be great to include those links you provided too 😄

**Comment by erindcole** [comment]
[MENTAL HUES LOGO_ideas_B.pdf](https://github.com/mentalhues/mentalhues.github.io/files/2153001/MENTAL.HUES.LOGO_ideas_B.pdf)

This is a set of logo ideas that @vanesachristman sent me. Wondering if we could start a discussion about these ideas, and maybe some others, for how the logo should look and what it needs to have to represent the community. Thoughts?

**Comment by yochannah** [comment]
the third one looks awesome! I also like the way the gradient represents changing / different perspectives, although I wonder if something rainbow-ey might be more representative of multiple different hues :) 

The book without the text would also make a nice logo for when space is small, and it might be nice to have a gradient-free black and white logo version that is suitable for printing.

**Comment by yochannah** [comment]
yes, good point! Not that it's a bad thing, but maybe not the intent. I
wonder if colours could be chosen to not intersect with a pride flag, maybe
- but more than 2 might be good.

**Comment by brennacole** [comment]
I would remove the book/any image all  together and make the font all lower case letters to make the logo seem more friendly.  The hues of green in the font are great.  Keep those.

**Comment by erindcole** [comment]
@brennacole are there other images you might suggest? @yochannah had a point that something like the book might be good for when the logo needs to be smaller, maybe for a favicon.

**Comment by vanesachristman** [comment]
Hi everyone thanks for the input! I'll try a gradient with more colors but not necessarily a rainbow, lowercase letters and ideas for a new icon. The initial idea for the book icon comes from the stories told in the website by the contributors: book of stories with pages in many hues. If you can think of cool ideas for icons please feel free to suggest. :-)
 regards
vanesa

**Comment by vanesachristman** [comment]
Hi everyone I'd like to submit my new versions of the logo based on everybody's suggestions. please let me know what you think.
Thanks,
Vanesa
[MENTAL HUES LOGO_ideas_C.pdf](https://github.com/mentalhues/mentalhues.github.io/files/2207075/MENTAL.HUES.LOGO_ideas_C.pdf)

**Comment by yochannah** [comment]
I like it - option B is really gorgeous! (They all are, but B is my favourite). Once one logo is settled on it might be nice to double check what it looks like in monochrome too (i.e. if it's printed in black and white will it look ok?) and if it looks odd create a flat / non-gradient version. might not be needed though.

**Comment by erindcole** [comment]
The colors in the new version are so beautiful. I wonder if tweaking the little icon above the 'u' might be a way to make the logo more distinct. The movement each of the versions captures of the little icon works great. @vanesachristman , what if the icon still incorporated the idea of little images- little circles, little papers... - but the overall image was a pen. Or if there is something that can convey the message of an individual being grouped in a community. This might help with @yochannah 's suggestion with the logo looking as compelling when there is more of a flat/non-gradient version.

**Comment by vanesachristman** [comment]
Hi everyone sorry about the wait I had some personal problems the past month.
I created a version of the last logo in gray scale so you can see how it translates in BW and also a new version where you can see a outline of a classic pen wi the dots as it was discussed on the previous commentary please let me know what you  think . thanks, V
[options C and D_Mental_Hues.pdf](https://github.com/mentalhues/mentalhues.github.io/files/2325600/options.C.and.D_Mental_Hues.pdf)
anesa

**Comment by brennacole** [comment]
Theses logos are really coming along. I prefer option a to option b. The
image detail emphasizes a hue, the concept imbeded in the name of the
project, as opposed to the pen, which would emphasize the narratives users
type. Depicting a pen might be confusing to someone who looks at the logo
for the first time without knowing details about the project. In essence,
option a is more user friendly.

**Comment by KirstieJane** [comment]
Ditto @brennacole - I like option (a) a lot!

I hope you're doing ok @vanesachristman and taking care of yourself. Thank you so much for helping to building mental hues 💖

**Comment by vanesachristman** [comment]
Thank you Brenna I agree :-)

**Comment by vanesachristman** [comment]
Thank you for the good thoughts Kristie, yes things are getting better :-)

**Comment by erindcole** [comment]
I'm trying to figure out how to add the logo to the project. Doing this creates some questions about trademarks and making sure the Mental Hues project remains unique and that people will be able to know when they are actually interacting with the correct organization and project when they contribute. @brennacole , @KirstieJane , @vanesachristman , do you have input on this? I've been mostly looking into ways to make unregistered trademarks for now. I don't think we need to do too much for brand preservation at this stage of the project. I do want to make sure the logo is protected when we start to use it.

**Comment by vanesachristman** [comment]
I agree that we don’t need to worry about registering for now I did send a couple of links on how to do it one of them seems to be very affordable I wanted to ask how I can share all of this on get hub if I need to create a new issue if there is one already there or if email is OK to communicate

**Comment by erindcole** [comment]
@vanesachristman I have been researching trademarks and am in the process of displaying the logo on the organization page for Mental Hues. To answer your question, I first want to make sure the proper license is in place. I am going to create a new folder in the Organization GitHub Page to showcase the logo and allow it to be shared. It will also be the place for future developments and changes to the logo over time.

I am thinking about using an MIT Open License, the same license for the Mental Hues project repo. What do you think? I am currently developing the folder. Please check it out when it is up and let me know if you would like changes. I'd like to attribute you as the original designer of the logo as  well. With the license I am considering, you would be welcome to use the logo however you choose, as well as anyone else, as is. I believe this will help continue to encourage the Mental Hues brand as an open source project.

**Comment by vanesachristman** [comment]
Hi Erin sorry it took me so long to reply. I was busy for a while and then had to re-figure out how to reply on Git-hub (embarrassing). In answer to your question; yes, I would love to be referred to as the original designer, were there any changes to the logo since my design?
As far as the license you seem to know more about it then I do so I can't really give you feedback except for I trust that you did good research on the matter and arrived at the best solution. I think I know which was the final logo, I will upload as you requested on issue #1.
